mindjunkies/payments/models.py

```
@@ -44,6 +44,36 @@
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="balance_history"
+    )
+    transaction = models.ForeignKey(
+        Transaction, on_delete=models.CASCADE, related_name="balance_entries",
+        null=True, blank=True
+    )
+    amount = models.DecimalField(max_digits=10, decimal_places=2)
+    previous_balance = models.DecimalField(max_digits=10, decimal_places=2)
+    new_balance = models.DecimalField(max_digits=10, decimal_places=2)
+    description = models.CharField(max_length=255)
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)